ETWAnalyzer/EventDump/DumpExceptions.cs

```diff
@@ -326,7 +337,10 @@ private void PrintByFileAndProcess(List<MatchData> matches)
         {
             foreach (var processExceptions in matches.GroupBy(x => x.Process))
             {
+                ModuleDefinition processModule = processExceptions.First().Module;
+                string moduleInfo = processModule!= null ? GetModuleString(processModule, true) : "";
                 ColorConsole.WriteEmbeddedColorLine($"[magenta]{processExceptions.Key.GetProcessWithId(UsePrettyProcessName)} {processExceptions.Key.StartStopTags}[/magenta] {(NoCmdLine ? String.Empty : processExceptions.Key.CommandLineNoExe)}", ConsoleColor.DarkCyan);
+                ColorConsole.WriteEmbeddedColorLine($"[red]{moduleInfo}[/red]");
```

Comment:
This will print a superflous newline if -smi is not present. You should print this only if -smi is present.